composer.json

```diff
@@ -24,8 +24,8 @@
         "ergebnis/composer-normalize": "^2.0",
         "friendsofphp/php-cs-fixer": "^2.17",
         "jangregor/phpstan-prophecy": "^0.8.0",
-        "phly/phly-event-dispatcher": "^1.0",
-        "phpactor/test-utils": "^1.1",
+        "phly/phly-event-dispatcher": "dev-php8",
```

Comment:
Just had a look on this package, what is this used for ? A simple implement of psr14 for tests purpose ?

Reply:
Yes - it's for testing here, but Phpactor requires it finally. I opened a PR to add PHP 8 support, but not sure it will get any attention, so could either fork or find another PSR implementation.